### ISSUE TITLE
how to update mmd_tools_helper for Blender 2.8+

### ISSUE THREAD

**Issue by Hogarth-MMD** [powroupi/blender_mmd_tools#336]
Can the great and wise @powroupi (or someone else) please tell me how to update mmd_tools_helper for Blender 2.8 and later versions of Blender? mmd_tools_helper has a primary panel in its __init__.py file and a subpanel in almost every module. The subpanels load fine in Blender 2.79b but not in later Blender versions. How do I fix this? Various people have been requesting this update from me.

**Comment by Hogarth-MMD** [comment]
Is there any possibility of someone giving me a clue about this issue? A technique to puzzle-solve it using a diagnostic print or by typing code into the Blender python console, for example?
Is mmd_tools successfully using subpanels loaded onto a main panel? If so, how does that work?
Any estimated time when @powroupi will be available to help with this issue? I hope that @powroupi is well and happy.

**Comment by Reatella** [comment]
Yes, I also hope to use mmd_tools_helper in 2.8+. When will it be updated？

**Comment by NaughtDZ** [comment]
> Yes, I also hope to use mmd_tools_helper in 2.8+. When will it be updated？

just use this,is working with my blender3.3 lts will:
https://github.com/UuuNyaa/blender_mmd_tools
